extras-requirements.txt

```diff
@@ -34,6 +34,9 @@ plotly~=5.4, <5.12.0
 # To support python 11, google-cloud-bigquery requires grpcio>=1.49.1, which is incompatible with the grpcio version
 # required by frames (because it upgrades protobuf from 3.x to 4.x, breaking binary compatibility)
 google-cloud-bigquery[pandas, bqstorage]==3.14.1
+google-cloud-storage==2.14.0
```

Comment:
please make sure your versions do not conflict with [Fix gcs recursive delete](https://github.com/mlrun/mlrun/pull/6193)